medic/medic-run.js

```diff
@@ -44,7 +44,9 @@ var MEDIC_BUILD_PREFIX        = "medic-cli-build";
 var DEFAULT_WINDOWS_VERSION   = "store";
 var WINDOWS_VERSION_CHOICES   = ["store", "store80", "phone"];
 var DEFAULT_TIMEOUT           = 600; // in seconds
-var SERVER_RESPONSE_TIMEOUT   = 3000; // in milliseconds
+var SERVER_RESPONSE_TIMEOUT   = 15000; // in milliseconds
+var MAX_NUMBER_OF_TRIES       = 3;
+var WAIT_TIME_FOR_CORDOVA_VM  = 15000; // in milliseconds
```

Comment:
Sorry, small nitpick again: this name doesn't describe what we're waiting for. Specifically, we're waiting for some time before retrying.